src/lib/adapters/Caching.js

```diff
@@ -1,18 +1,25 @@
 import * as Tree from '../Tree'
+import { Folder, Bookmark } from '../Tree'
 import Adapter from '../Adapter'
 
 export default class CachingAdapter extends Adapter {
   constructor(server) {
     super()
     this.highestId = 0
+    this.bookmarksCache = new Folder ({id: 0, title: 'root'});
+  }
+
+  getLabel() {
+    let data = this.getData()
+    return data.username + '@' + data.url
   }
 
   async getBookmarksTree() {
     return this.bookmarksCache.clone()
   }
 
   async createBookmark(bm) {
-    console.log('(fake)CREATE', bm)
```

Comment:
could you replace the `(fake)` with `(webdav)`, so that the logs reflect which adapter is used?

Reply:
oh, right, this is not the webdav adapter.... then let's leave it like that.